python/inputmodule/gui/ledmatrix.py

```diff
@@ -57,7 +63,9 @@ def blinking(dev):
 
 def random_eq(dev):
     """Display an equlizer looking animation with random values."""
-    while True:
+    animate(dev, False)
+    set_status('random_eq')
+    while True and get_status() == 'random_eq':
```

Comment:
```suggestion
    while get_status() == 'random_eq':
```
More nitpicking :wink: